src/biogeochem/CNVegStructUpdateMod.F90

```diff
@@ -210,14 +216,28 @@ subroutine CNVegStructUpdate(num_soilp, filter_soilp, &
                   !correct height calculation if doing accelerated spinup
                   if (spinup_state == 2) then
                     htop(p) = ((3._r8 * deadstemc(p) * 10._r8 * taper * taper)/ &
-                         (SHR_CONST_PI * stocking * dwood(ivt(p))))**(1._r8/3._r8)
+                         (SHR_CONST_PI * nstem(ivt(p)) * dwood(ivt(p))))**(1._r8/3._r8)
                   else
                     htop(p) = ((3._r8 * deadstemc(p) * taper * taper)/ &
-                         (SHR_CONST_PI * stocking * dwood(ivt(p))))**(1._r8/3._r8)
+                         (SHR_CONST_PI * nstem(ivt(p)) * dwood(ivt(p))))**(1._r8/3._r8)
                   end if
 
                endif
 
+               ! calculate vegetation physiological parameters used in biomass heat storage
+               if (use_biomass_heat_storage) then
+                  ! Assumes fbw the same for leaves and stems
+                  leaf_biomass(p) = max(0.0025_r8,leafc(p)) &
+                       * c_to_b * 1.e-3_r8 / (1._r8 - fbw(ivt(p)))
+
+                  stem_biomass(p) = (deadstemc(p) + livestemc(p)) &
+                       * c_to_b * 1.e-3_r8 / (1._r8 - fbw(ivt(p)))
+                  
+                  if (spinup_state == 2) then
+                     stem_biomass(p) = 10._r8 * stem_biomass(p)
```

Comment:
if (spinup_state == 2) then
    stem_biomass(p) = (10._r8 * deadstemc(p) + livestemc(p)) &
                           * c_to_b * 1.e-3_r8 / (1._r8 - fbw(ivt(p)))

Reply:
@ekluzek this sounds nice to make the 10._r8 a constant if it's not too time consuming

Given @dlawrenncar 's comment, it seems like replacing the statement after the logical `spinup_state == 2` would be more accurate  (see above)

Reply:
OK, this has been done.